mantidproject/mantid
# Add a sector shape to the instrument viewer

**Description of work.**
This PR adds a circular sector shape in the Pick and Mask tabs of the instrument viewer. Since this shape is based on a circle, it cannot be freely deformed, so scaling is a bit different than what it is for the other shapes.

**To test:**
 - Open a nexus file with an instrument associated (e.g. `ExternalData/Testing/Data/UnitTest/ILL/PANTHER/002687.nxs` ).
- Open the instrument view.
- Go to the Pick Tab.
- Create a sector on the view.
- Modify it, changing the bounding box, the inner and outer radius, the angles etc ... and check if the behavior is what you would expect.
- Check the same on the Mask tab.
- Save and re-load the project


Fixes #28352  

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
